sdk/identity/identity/test/node/managedIdentityCredential.spec.ts

```diff
@@ -132,6 +134,44 @@ describe("ManagedIdentityCredential", function() {
   //   assert.strictEqual(mockHttpClient.requests.length, 1);
   // });
 
+  it("sends an authorization request correctly in an App Service environment version 2019-08-01", async () => {
+    // Trigger App Service behavior by setting environment variables
+    process.env.IDENTITY_ENDPOINT = "https://endpoint";
+    process.env.IDENTITY_HEADER = "secret";
+
+    const authDetails = await getMsiTokenAuthRequest(["https://service/.default"], "client", {
+      authResponse: {
+        status: 200,
+        parsedBody: {
+          token: "token",
+          expires_on: "1560999478"
```

Comment:
If I'm understanding this correctly, the server will now answer with a seconds since epoch string, like `"1560999478"`.